kernel/src/process/exit.rs

```diff
@@ -15,6 +15,7 @@ use crate::{prelude::*, process::signal::signals::kernel::KernelSignal};
 pub(super) fn exit_process(thread_local: &ThreadLocal, current_process: &Process) {
     current_process.status().set_zombie();
 
+    current_process.status().set_vm_shared_status(false);
```

Comment:
This has nothing to do with the file table. Why put this line here? Moving it closer to `status().set_zombie()` actually makes some sense.